docs/superpowers/specs/2026-04-15-phase1-plumbing-design.md

```
@@ -0,0 +1,461 @@
+
+export function trackEvent(event: string, properties?: Record<string, unknown>): void {
+  posthog.capture(event, properties);
+}
+```
+
+### Design decisions
+
+- **Conditional init on `api_key`.** PostHog infra (wcpos/wcpos-infra#18) may not be deployed when this ships. Empty `api_key` = skip init entirely.
+- **`persistence: 'memory'`** — no cookies or localStorage in wp-admin context.
+- **`autocapture: false`** — wp-admin has too many irrelevant elements. Explicit tracking only.
+- **`trackEvent  helper** — thin wrapper so components don't import posthog directly. If PostHog isn't initialized, `posthog.capture` is a no-op.
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)